include/mme-app/actionHandlers/actionHandlers.h

```diff
@@ -213,6 +218,11 @@ namespace mme
         ***********************************************/
         static SM::ActStatus erab_mod_ind_complete(SM::ControlBlock& cb);                
 
+        /**********************************************
+        * Action handler : fwd_rel_req_handler
+        ***********************************************/
+        static SM::ActStatus fwd_rel_req_handler(SM::ControlBlock& cb);                
```

Comment:
I don't see forward relocation related structures defined in msgType file.

Reply:
s10_struct.h file is empty. You wanted to add something there ?

Reply:
We have added required code in the  mentioned files. We will raise PR request for the updated code very soon.